src/database.cpp

```diff
@@ -200,6 +201,9 @@ std::string Database::open(bool wait, bool memory) {
       const char *set_version = "insert or ignore into schema(version) values(" SCHEMA_VERSION ");";
       ret = sqlite3_exec(imp->db, get_version, &schema_cb, 0, 0);
       if (ret == SQLITE_OK) {
+        if (waiting) {
+          std::cerr << std::endl;
+        }
```

Comment:
What motivates this print statement?

Reply:
I think this needs to be move to the top of the enclosing 'of'; ie: right before the "use an empty entropy table" comment.

Reply:
Done.